packages/ui-codemods/lib/utils/codemodHelpers.ts

```diff
@@ -503,6 +503,62 @@ function addImportIfNeeded(
   return name
 }
 
+/**
+ * Renames an import and all its usages throughout the file. Modifies the input collection.
+ * Handles both direct imports and renamed imports (with 'as' syntax).
+ * ❗ Does NOT support default imports like `import Button from './Button'`
+ * @example
+ * renameImportAndUsages(j, root, path, 'oldName', 'newName')
+ *
+ * replaces
+ * ```
+ * import { oldName } from "module"
+ * import { oldName as localName } from "module"
+ * oldName()
+ * localName()
+ * ```
+ * with
+ * ```
+ * import { newName } from "module"
+ * import { newName as localName } from "module"
+ * newName()
+ * localName()
+ * ```
+ *
+ * @param j - jscodeshift API
+ * @param root - Root AST collection
+ * @param oldName - Original imported name to replace
+ * @param newName - New name to use
+ * @returns boolean indicating if any changes were made
+ */
+function renameImportAndUsages(
+  j: JSCodeshift,
+  root: Collection,
+  oldName: string,
+  newName: string
+): boolean {
```

Comment:
For this to be a good utility function one should also specify the import path. While your use case here (import names like `canvasThemeLocal`) are pretty much guaranteed to be unique, if we want to rename e.g. `Button` with this then I bet that it would make lots of false renames. Also this new param should work like in `replaceImport`, meaning one should be able to specify either a string or an array of strings